modules/serverless-rn-1-34-1.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-34-1_{context}"]
+= Red Hat {ServerlessProductName} 1.34.1
+
+{ServerlessProductName} 1.34.1 is now available. This release of {ServerlessProductName} addresses identified Common Vulnerabilities and Exposures (CVEs). Fixed issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes:
+
+[id="fixed-issues-1-34-1_{context}"]
+== Fixed issues
+
+* Previously, requests to Knative Services with slow back-end responses could fail due to the default OpenShift Route timeout being too short. This issue has been addressed by making the `haproxy.router.openshift.io/timeout` setting configurable for automatically created routes.
++
+You can now adjust the timeout by setting the `ROUTE_HAPROXY_TIMEOUT` environment variable in the {ServerlessOperatorName} Subscription configuration as follows:
```

Comment:
```suggestion
You can now adjust the timeout by setting the `ROUTE_HAPROXY_TIMEOUT` environment variable in the {ServerlessOperatorName} subscription configuration as follows:
```

OR `Subscription` (with back ticks) configuration

Reply:
Added with the `Subscription` back ticks.